azure-stack/hci/upgrade/install-enable-network-atc.md

```diff
@@ -51,21 +51,21 @@ In this step, you install Network ATC on every node in the cluster using the fol
 Install-WindowsFeature -Name NetworkATC
 ```
 
-### Step 2: Pause one node in the cluster
+### Step 2: Stop the Network ATC service
 
-When you pause one node in the cluster, all workloads are moved to other nodes, making your machine available for changes. The paused node is then migrated to Network ATC. To pause your cluster node, use the following command:
+To prevent Network ATC from applying the intent while VMs are running, stop or disable the Network ATC service on all nodes that aren't paused. Use these commands:
 
 ```powershell
-Suspend-ClusterNode
+Set-Service -Name NetworkATC -StartupType Disabled
+Stop-Service -Name NetworkATC
 ```
 
-### Step 3: Stop the Network ATC service
+### Step 3: Pause one node in the cluster
```

Comment:
@helmste - I will need to check with the PM on switching this step.

Reply:
same as above